src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts

```diff
@@ -1008,7 +1008,7 @@ export class ReplicationWizardComponent implements WizardConfiguration {
             failedToLoadChildren: true,
           });
           const sourceDatasetsFieldConfig = _.find(this.wizardConfig[0].fieldConfig, { name: 'source_datasets' });
-          sourceDatasetsFieldConfig.warnings = 'Failed to load sub-directories/datasets';
+          sourceDatasetsFieldConfig.warnings = T('Failed to load sub-directories/datasets');
```

Comment:
If I understand the context correctly, here we can only have datasets, not directories.